lib/handlebars/helpers/if.js

```diff
@@ -1,7 +1,9 @@
-import {isEmpty, isFunction} from '../utils';
+import { isEmpty, isFunction } from '../utils';
+import Exception from '../exception';
 
 export default function(instance) {
   instance.registerHelper('if', function(conditional, options) {
+    if (arguments.length != 2) { throw new Exception('#if requires exactly one argument');}
```

Comment:
This really look weird. (`length != 2` leads to a message with `requires exactly one...`), but it is correct.